internal/scanner/scanner.go

```diff
@@ -257,7 +257,7 @@ func (s *Scanner) ScanClusterWideResources(ctx context.Context, runUID string) e
 			Int("parallel-resources-audits", s.parallelResourcesAudits),
 		).Msg("cluster admission policies count")
 
-	for gvr, policies := range policies.PoliciesByGVR {
+	for gvr, policiesSlice := range policies.PoliciesByGVR {
```

Comment:
Personally, I would stick with the old name, as idiomatic go does not use Hungarian notation and I find it good enough.

Reply:
This is to not confuse the `policies *Policies{}` with the `policies []Policy` , that is already selected and gets used in the goroutine later on. If not, both identifiers clash.

I could rename `policiesSlice` to `pols` to just use inside the for.

It could be possible to pass `policies *Policies{}` , `resourceGVR` to `auditResource()`, and that way we would have also `policies.Skipped` and so on. But then we are passing the whole hashmap of policies on each `auditResource()` call, incrementing the mem usage.

Reply:
Renamed to `pols`.